libp2p/peer/peermetadata_interface.py

```diff
@@ -1,13 +1,20 @@
 from abc import ABC, abstractmethod
+from typing import (
+    Any,
+)
+
+from .id import (
+    ID,
+)
 
 
 class IPeerMetadata(ABC):
 
-    def __init__(self):
+    def __init__(self) -> None:
         pass
 
     @abstractmethod
-    def get(self, peer_id, key):
+    def get(self, peer_id: ID, key: Any) -> Any:
```

Comment:
https://github.com/libp2p/go-libp2p-core/blob/master/peerstore/peerstore.go#L80-L86
```suggestion
    def get(self, peer_id: ID, key: str) -> Any:
```